postgres_to_redshift.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"strings"
+
+	"github.com/clever/redshifter/postgres"
+	"github.com/clever/redshifter/redshift"
+	"github.com/segmentio/go-env"
+)
+
+var (
+	// TODO: include flag validation
+	awsRegion  = env.MustGet("AWS_REGION")
+	s3prefix   = flag.String("s3prefix", "", "s3 path to be used as a prefix for temporary storage of postgres data")
+	tables_csv = flag.String("tables", "", "Tables to copy as CSV")
+	dumppg     = flag.Bool("dumppostgres", true, "Whether to dump postgres")
+	updateRS   = flag.Bool("updateredshift", true, "Whether to replace redshift")
```

Comment:
@nathanleiby I prefer flags wherever possible because the go support for them is much better. The only reason AWS key, secret and region are from ENV is because these are used by the pathio library so it makes sense to re-use the same here.